src/ExpressionEvaluator/CSharp/Source/ExpressionCompiler/Binders/EEMethodBinder.cs

```diff
@@ -45,7 +45,7 @@ internal EEMethodBinder(EEMethodSymbol method, MethodSymbol containingMethod, Bi
             _parameterOffset = substitutedSourceMethod.IsStatic ? 0 : 1;
             _targetParameters = method.Parameters;
             // https://github.com/dotnet/roslyn/issues/62334: add tests and adjust implementation to allow EE to access file types
-            _sourceBinder = new InMethodBinder(substitutedSourceMethod, new BuckStopsHereBinder(next.Compilation, associatedSyntaxTree: null));
+            _sourceBinder = new InMethodBinder(substitutedSourceMethod, new BuckStopsHereBinder(next.Compilation, associatedFileIdentifier: null));
```

Comment:
>null [](http://example.com/codeflow?start=140&length=4)

Why `null` is always the right value to pass here?

Reply:
This binder is only used to call 'LookupSymbolsInSingleBinder', and we never expect this binder to look for symbols that could be file-local.

Reply:
Consider adding a comment